python-squid/src/test/java/org/sonar/python/parser/python_v3/PythonV3Test.java

```diff
@@ -46,6 +46,20 @@ public void function_declaration(){
   }
 
   @Test
+  public void asyncAndAwait() {
+    setRootRule(PythonGrammar.FUNCDEF);
+    assertThat(p).matches("async def fun(): pass");
```

Comment:
@vilchik-elena I remember that I tried, but I failed due to the newline char between the annotation and the rest of the function declaration. I don't know how to account for the newline